src/content/docs/logs/get-started/live-archives.mdx

```diff
@@ -139,13 +123,9 @@ There are two ways to query live archives: with Lucene or NRQL. If you want to u
    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Logs**</DoNotTranslate> to query data stored with live archives.
 </figcaption>
 
-</Step>
-
-
-<Step>
 ### Query live archives with NRQL [#nrql]
 
-If you want to query using the same language you use for errors inbox or alerts, we suggest querying in NRQL. You'll need the **Live archives query** capability to query live archives.
+If you want to query using the same language you use for errors inbox or alerts, we suggest querying in NRQL. 
```

Comment:
@homelessbirds are you missing a step? When users just go to "Logs" it defaults to the Lucene query page, right? Don't you need to tell users to click **NRQL**?